mojo/private/mojo_binary_test.bzl

```diff
@@ -126,10 +129,14 @@ def _mojo_binary_test_implementation(ctx, *, shared_library = False):
             "MODULAR_MOJO_MAX_COMPILERRT_PATH": "/dev/null",  # Make sure this fails if accessed
             "MODULAR_MOJO_MAX_LINKER_DRIVER": "/dev/null",  # Make sure this fails if accessed
             "MODULAR_MOJO_MAX_LLD_PATH": mojo_toolchain.lld.path,
+            # "MODULAR_MOJO_MAX_LLD_PATH": "/dev/null",  # Make sure this fails if accessed
```

Comment:
this doesn't work because this path isn't mapped, this is only accessed in some cases